src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -78,10 +82,31 @@ public object Model
         /// <summary>
         /// When optional, executing the tag helper will no-op if the view cannot be located. 
         /// Otherwise will throw stating the view could not be found.
+        /// Cannot be used together with <see cref="Fallback"/>.
         /// </summary>
         [HtmlAttributeName(OptionalAttributeName)]
         public bool Optional { get; set; }
 
+        /// <summary>
+        /// View to lookup if main view cannot be located.
+        /// Will throw if main view and fallback cannot be located.
+        /// Cannot be used together with <see cref="Optional"/>.
+        /// </summary>
+        [HtmlAttributeName(FallbackAttributeName)]
+        public string Fallback
+        {
+            get => _fallback;
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                {
+                    throw new ArgumentNullException(nameof(Fallback));
+                }
+                _fallback = value;
+                _hasFallback = true;
```

Comment:
Do you need this field? Could you just use `IsNullOrEmpty` during execution to determine if this was set?